exercism/bracket-push/bracket-push.js

```diff
@@ -0,0 +1,44 @@
+function bracket(input) {
+  var indexArray = input.split('')
+  var valueArray = []
+
+  for (var i = 0; i < indexArray.length; i++) {
+    if (indexArray[i] === '(') {
+      valueArray.push(7);
+    }
+    if (indexArray[i] === ')') {
+      valueArray.push(-7);
+    }
+    if (indexArray[i] === '{') {
+      valueArray.push(5);
+    }
+    if (indexArray[i] === '}') {
+      valueArray.push(-5);
+    }
+    if (indexArray[i] === '[') {
+      valueArray.push(2)
+    }
+    if (indexArray[i] === ']') {
+      valueArray.push(-2)
+    }
+  }
+  for (var j = 0; j < valueArray.length; j++) {
+    if (valueArray[j] + valueArray[j + 1] === 0 && valueArray[j] > valueArray[j + 1]) {
```

Comment:
This is such a cool way to go about doing it lol. I would have never though of it.